src/lists/list-item.jsx

```diff
@@ -131,12 +132,13 @@ const ListItem = React.createClass({
     const twoLine = secondaryText && secondaryTextLines === 1;
     const threeLine = secondaryText && secondaryTextLines > 1;
     const hasCheckbox = leftCheckbox || rightToggle;
+    const selectedColor = hoverColor;
 
     const styles = {
       root: {
-        backgroundColor: (this.state.isKeyboardFocused || this.state.hovered) &&
+        backgroundColor: (this.props.selected && !this.state.hovered) ? selectedColor : ((this.state.isKeyboardFocused || this.state.hovered) &&
```

Comment:
the `...other` is going to have the `selected` property then passed to the `EnhancedButton`. Is this intentional? Otherwise, can you remove it.